language/cypher/src/main/java/com/neueda/jetbrains/plugin/graphdb/language/cypher/editor/TypedHandlerDelegate.java

```diff
@@ -0,0 +1,27 @@
+package com.neueda.jetbrains.plugin.graphdb.language.cypher.editor;
+
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.neueda.jetbrains.plugin.graphdb.language.cypher.psi.CypherStringLiteral;
+import org.jetbrains.annotations.NotNull;
+
+public class TypedHandlerDelegate extends com.intellij.codeInsight.editorActions.TypedHandlerDelegate {
```

Comment:
Prefix with `Cypher` and properly import base class

Reply:
resolved in [16aaf53](https://github.com/neueda/jetbrains-plugin-graph-database-support/pull/109/commits/16aaf537f5abce64f4861eaf49922a760e38e37f)